marklogic-data-hub-central/ui/src/components/monitor-selected-facets/monitor-selected-facets.tsx

```diff
@@ -81,7 +82,7 @@ export const MonitorSelectedFacets: (React.FC<Props>)  = (props) => {
             data-testid={`clear-${item.facet}`}
           >
             {displayName}
-            <Icon type="close"/>
+            <XLg className={styles.close}/>
```

Comment:
@federicoRufrancosMarklogic, can you please fix the following in the "Monitor Flows and Steps" view:
1. Decrease padding (slightly) around the facet close icon, to the left of the facet label, and on the top of the facet container so that they look more like the current version. (Do this for the unselected and selected versions of the facet container.)
2. Change the background color for the selected version from dark blue to white (or clear).
3. When there is no pagination on the page (to get this, make a facet selection that gives no results), there is no space between the title and the selected tags. Add some space between the "Monitor Flows and Steps" title and the selected tags to account for this. 
![7853](https://user-images.githubusercontent.com/477757/134603744-c423c4e3-9827-45a6-b6f6-eade78720f71.png)
Let me know if you have questions!

Reply:
I think I got all the topics, when every you got a minute could you double check the fixes @wooldridge ?